<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="selectpage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="selectpage1">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>448</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="firstbutton1">
        <property name="text">
         <string>首页</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stockbutton1">
        <property name="text">
         <string>股票</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankbutton1">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dealbutton1">
        <property name="text">
         <string>交易规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="communitybutton1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>社区交流</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="selectpage2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>448</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="firstbutton2">
        <property name="text">
         <string>首页</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stockbutton2">
        <property name="text">
         <string>股票</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankbutton2">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dealbutton2">
        <property name="text">
         <string>交易规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="communitybutton2">
        <property name="text">
         <string>社区交流</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="selectpage3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>448</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="firstbutton3">
        <property name="text">
         <string>首页</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stockbutton3">
        <property name="text">
         <string>股票</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankbutton3">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dealbutton3">
        <property name="text">
         <string>交易规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="communitybutton3">
        <property name="text">
         <string>社区交流</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="selectpage4">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>448</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="firstbutton4">
        <property name="text">
         <string>首页</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stockbutton4">
        <property name="text">
         <string>股票</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankbutton4">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dealbutton4">
        <property name="text">
         <string>交易规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="communitybutton4">
        <property name="text">
         <string>社区交流</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="selectpage5">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>448</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="firstbutton5">
        <property name="text">
         <string>首页</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stockbutton5">
        <property name="text">
         <string>股票</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rankbutton5">
        <property name="text">
         <string>排行榜</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dealbutton5">
        <property name="text">
         <string>交易规则</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="communitybutton5">
        <property name="text">
         <string>社区交流</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
